### PR TITLE
Add CocoaPods support

### PR DESCRIPTION
**Pull request by dnkoutso**
cc @mattgallagher @mseijas

Those will require to be published to cocoapods trunk via `pod trunk push` if this is accepted and merged.

### CODE REVIEW

File: CwlCatchExceptionSupport.podspec [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Pod::Spec.new do |s|
+  s.name         = 'CwlCatchExceptionSupport'
+  s.version      = '2.1.0'
+  s.summary      = 'TODO'
```
Comment (dnkoutso) [inline CwlCatchExceptionSupport.podspec:4]: what is the short description to use for this?

**Comment by mseijas** [comment]
The internal dependencies need to be pushed first before `CwlCatchException`, so they would need to be published in this order:

```bash
pod trunk push CwlCatchExceptionSupport.podspec
pod trunk push CwlCatchException.podspec
```

**Comment by mattgallagher** [comment]
I've merged this but it is not yet pushed to Cocoapods since I'm getting internal server errors trying to get a new authentication token. Will give it a few hours and try again.

**Comment by mattgallagher** [comment]
Okay, this should be sorted now. The version is bumped to 2.1.1 to pick up the `#if SWIFT_PACKAGE || COCOAPODS` change.

**Pull request merged**